etc/testing/circle/run_tests.sh

```diff
@@ -83,13 +83,16 @@ case "${BUCKET}" in
         make test-tls
     fi
     ;;
+  INTERNAL)
+    go install -v ./src/testing/match
```

Comment:
Maybe wrap this in a make task so it can be run from make test? (Not sure how common running tests locally is)

Reply:
I don't think that running the entire test suite locally is very common, and I'm trying to move away from adding Make targets in favour of straight `go test` targets. If people feel strongly we can re-add the Make target but I think it just abstracts and causes these kinds of issues.

Reply:
Makes sense, mostly proposing for consistency, but agree with the move away from make targets. I'd love to be able to run `go test ./...` on the whole repo